ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/manager/EthSchedulerTest.java

```diff
@@ -205,7 +205,6 @@ public void timeout_cancelsTaskWhenResultIsCancelled() {
     result.cancel(false);
     assertThat(task.isDone()).isTrue();
     assertThat(task.isFailed()).isTrue();
-    assertThat(task.isSucceeded()).isFalse();
```

Comment:
I'm not sure why we should remove isSucceeded but we should keep isFailed. maybe I missed something

Reply:
I removed the method entirely and here because this was the only place it was used and if the previous two assertions, `isDone` and `isFailed` are true, `isSucceeded` can never be true.